### ISSUE TITLE
Imperfect looks on the Color controls

### ISSUE THREAD

**Issue by Jay-o-Way** [microsoft/WinUI-Gallery#1561]
### Which version of the app?

WinUI 3 Gallery

### Description

Minor issue.

- In some occasions, it looks like the border of the **ColorTile** control and the child element(s) do not have the same CornerRadius, or a padding (?) that creates a border.
- In some occasions, it looks like 4px is used.
- The border that acts as a separator is used literally only **one** time (at AcrylicBackgroundFillColorBaseBrush) and it isn't even visible! Maybe we should just remove it.
- In some (2) places, a border with `CornerRadius="4"` is used. not only is it hard-coded, it's also the incorrect value for the cases.

### Screenshots

![Schermafbeelding 2024-06-13 150625](https://github.com/microsoft/WinUI-Gallery/assets/65828559/a2951a2b-9e31-4624-b2ab-bb36c2bb300d)

![Schermafbeelding 2024-06-13 152029](https://github.com/microsoft/WinUI-Gallery/assets/65828559/cbdf405a-ec79-4c6e-b571-6c18ffe48b71)

> 4px?

**Comment by niels9001** [comment]
@jay-o-way Something you'd like to pick up?

**Comment by Jay-o-Way** [comment]
I can. Just have to see when. Next week is my vacation (incl Graspop🤘🏻)

**Comment by Jay-o-Way** [comment]
@niels9001 I think this line is unused and can be removed, and then the two Grids can be merged to one. Can you confirm?
https://github.com/microsoft/WinUI-Gallery/blob/6be8d2d8c1d647f0bacee437ba77326450ef5e39/WinUIGallery/Controls/DesignGuidance/ColorPageExample.xaml#L15

**Comment by Jay-o-Way** [comment]
Style "GalleryTileGridStyle" uses `OverlayCornerRadius`, but the following Grids have 4px corners, and at the moment I can't find out why these are exceptions. #needhelp

- SmokeFillColorDefaultBrush, AccentAcrylicBackgroundFillColorDefaultBrush
- Text, AccentText
- AccentFill
- Controlstrong

**Comment by niels9001** [comment]
> @niels9001 I think these lines are unused and can be removed, which means the two top Grids can be removed. Can you confirm?
> 
> https://github.com/microsoft/WinUI-Gallery/blob/6be8d2d8c1d647f0bacee437ba77326450ef5e39/WinUIGallery/Controls/DesignGuidance/ColorPageExample.xaml#L15

Yes, even the root grid can be removed here?
> 
> https://github.com/microsoft/WinUI-Gallery/blob/6be8d2d8c1d647f0bacee437ba77326450ef5e39/WinUIGallery/Controls/DesignGuidance/ColorTile.xaml#L17

I think we are using that columndefinition for the border?

**Comment by niels9001** [comment]
> > I think we are using that columndefinition for the border?
> 
> Ah yes, the **ShowSeparator**. Thing is - in practice it isn't used, so I already removed it (locally) when working on this. Therefore the ColumnDefinition is obsolete.

Just looking at the code.. it's turned on by default:

![image](https://github.com/user-attachments/assets/9cb6bd31-4850-4777-8e05-74357d56eefb)

And in 8 instances we turn it off https://github.com/search?q=repo%3Amicrosoft%2FWinUI-Gallery%20ShowSeparator&type=code?

So I think we are using it?

**Comment by Jay-o-Way** [comment]
@niels9001 No, it is not used. Well, technically in two occasions, but it's invisible!
![image](https://github.com/user-attachments/assets/e9a93840-50bc-4b38-bb91-0972dbf55340)

**Comment by niels9001** [comment]
@jay-o-way It is used in some occasions, and it is visible for me?

Set to true:
![image](https://github.com/user-attachments/assets/6aa007cd-91e8-4c92-9957-d7b20c7fd9e8)

Set to false:
![image](https://github.com/user-attachments/assets/e4b700d7-2215-41c2-8b67-c689d7988d8c)

I think, since we are use it here and there to make the default be `false` and only turn it on when needed to reduce the verbosity of XAML. But other than that, I think we need to keep it..

**Comment by Jay-o-Way** [comment]
I see. Focussed on the declared ("false") values and missed the instances where it wasn't declared. 🫣
We could elevate this topic to a higher level: why not choose to use it either always or never?

**Comment by niels9001** [comment]
> I see. Focussed on the declared ("false") values and missed the instances where it wasn't declared. 🫣 We could elevate this topic to a higher level: why not choose to use it either always or never?

This was per design spec. We only need it when 2 neighboring colors are similar to provide the right contrast. I don't think there's a problem here that needs to be solves? Having it default to `false` would be my only proposed change.

On the CornerRadius issues - yes, we should remove any hardcoded 4px radiuses if those are defined. Not sure why those were defined in the first place though 😨

**Comment by Jay-o-Way** [comment]
I've been digging into this now, but the corners are weird. Looks like it's not a matter of a property, but more of the rendering...
